### ISSUE TITLE
Navidrome cant play any Song from this album

### ISSUE THREAD

**Issue by quotengrote** [deluan/navidrome#730]
Hi, 

thx for your great Work.

I have an Navidrome-Instance within Docker.
It works great but this album cant be played:
- https://mgrote.next-cloud.org/index.php/s/q9PQo829GQt8gE2

## Docker-Compose: 
```
version: '3'
services:
######## navidrome-mg ########
  navidrome-mg:
    container_name: "navidrome-mg"
    image: deluan/navidrome:latest
    restart: always
    environment:
      - ND_SCANINTERVAL=15m
      - ND_LOGLEVEL=info
      - ND_SESSIONTIMEOUT=24h
      - ND_MUSICFOLDER=/music
      - ND_DATAFOLDER=/data
      - ND_TRANSCODINGCACHESIZE=500MB
      - ND_IMAGECACHESIZE=100MB
      - ND_AUTOIMPORTPLAYLISTS=false
#      - ND_UILOGINBACKGROUNDURL
#      - ND_LASTFM_APIKEY=766f890f25df0c4d6888f8a20f6402af
#      - ND_LASTFM_SECRET=793bb020c96eb438e32f166e2416d22d
      - ND_LASTFM_LANGUAGE="de"
      - ND_BASEURL=/mg
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Berlin
    volumes:
      - musik_smb_mg:/music:ro
      - data:/data
    networks:
      - proxy
    labels:
      - com.centurylinklabs.watchtower.enable=true
      - traefik.http.routers.navidrome-mg.rule=Host(`audio.mXXXXXXXX.net`)&&PathPrefix(`/mg`)
      - traefik.enable=true
      - traefik.http.routers.navidrome-mg.tls=true
      - traefik.http.routers.navidrome-mg.tls.certresolver=resolver_letsencrypt
      - traefik.http.routers.navidrome-mg.entrypoints=websecure
      - traefik.http.services.navidrome-mg.loadbalancer.server.port=4533
    ports:
      - "4533:4533"


######## Volumes ########
volumes:
  data:
  musik_smb_mg: # Verzeichnis in restic aus Backup ausnehmen!
    driver: local
    driver_opts:
      type: "cifs"
      o: "user=navidrome,password=XXXXXXXX"
      device: "//192.168.2.36/musik/Musik"
######## Networks ########
networks:
  proxy:
    external: true

```

**Comment by metalheim** [comment]
You probably should delete your Last.fm secret from your message.

Sorry that I'm not able to contribute further to your issue.

**Comment by andyklimczak** [comment]
Try opening the album in something like easytag, edit, and resave the metadata. I had a few albums where the library navidrome uses to get metadata errors out when reading some files from less than reputable sources

**Comment by quotengrote** [comment]
> You probably should delete your Last.fm secret from your message.
Thx for the reminder, but these are only my test Credentials :-)


> Try opening the album in something like easytag, edit, and resave the metadata. I had a few albums where the library navidrome uses to get metadata errors out when reading some files from less than reputable sources
Did that with MediaMonkey, did not work.

**Comment by andyklimczak** [comment]
If you change the `LogLevel` to [`TRACE`](https://www.navidrome.org/docs/usage/configuration-options/#configuration-file), do any errors get printed when you try to play the file?

**Comment by certuna** [comment]
Unfortunately that download link doesn't seem to work?

**Comment by quotengrote** [comment]
> If you change the [`LogLevel` to `TRACE`](https://www.navidrome.org/docs/usage/configuration-options/#configuration-file) or `DEBUG`, do any related errors get printed when you try to play the file?

Thats the output with level "Debug", i dont see anything special.
```time="2021-01-21T17:54:25Z" level=debug msg="API: Successful response" requestId=3174dffac93a/GGPbJYGfPS-000046 status=OK
time="2021-01-21T17:54:25Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/rest/scrobble?u=mg&t=30f3daf29af6381329d5e6abd40837c4&s=373128&f=json&v=1.8.0&c=NavidromeUI&id=761cbb01614005c2d93af911d3f83499&submission=false" elapsedTime=1.457581ms httpStatus=200 remoteAddr=93.200.208.10 requestId=3174dffac93a/GGPbJYGfPS-000046 responseSize=110 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:54:30Z" level=debug msg="API: New request /mg/rest/stream" client=NavidromeUI requestId=3174dffac93a/GGPbJYGfPS-000047 username=mg version=1.8.0
time="2021-01-21T17:54:30Z" level=debug msg="Streaming RAW file" id=9fad2d65ebd858a51ec0c9f735d3e5c4 originalBitrate=830 originalFormat=flac path="/music/W/Wintergatan - Wintergatan/Slottskogen Disc Golf Club_Z4434.flac" requestBitrate=0 requestFormat= requestId=3174dffac93a/GGPbJYGfPS-000047 selectedBitrate=0 selectedFormat=raw
time="2021-01-21T17:54:30Z" level=info msg="Streaming file" artist=Wintergatan bitRate=0 cached=false format=raw originalBitRate=830 originalFormat=flac title="Slottskogen Disc Golf Club" transcoding=false user=mg
time="2021-01-21T17:54:31Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/rest/stream?u=mg&t=30f3daf29af6381329d5e6abd40837c4&s=373128&f=json&v=1.8.0&c=NavidromeUI&id=9fad2d65ebd858a51ec0c9f735d3e5c4&_=1611251673634" elapsedTime=878.77217ms httpStatus=206 remoteAddr=93.200.208.10 requestId=3174dffac93a/GGPbJYGfPS-000047 responseSize=14566944 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:54:46Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/app/api/song/761cbb01614005c2d93af911d3f83499" elapsedTime="606.414µs" httpStatus=200 remoteAddr=93.200.208.10 requestId=3174dffac93a/GGPbJYGfPS-000049 responseSize=1144 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
```

same with level=trace
```
time="2021-01-21T17:58:00Z" level=info msg="Now Playing" artist="Belle and Sebastian" title="I Didn't See It Coming" user=mg
time="2021-01-21T17:58:00Z" level=debug msg="API: Successful response" body="{\"subsonic-response\":{\"status\":\"ok\",\"version\":\"1.16.1\",\"type\":\"navidrome\",\"serverVersion\":\"0.39.0 (f2a8308)\"}}" requestId=fe918e0edfb1/CBFxgx1qQ9-000022 status=OK
time="2021-01-21T17:58:00Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/rest/scrobble?u=mg&t=30f3daf29af6381329d5e6abd40837c4&s=373128&f=json&v=1.8.0&c=NavidromeUI&id=636eed0b1b989c5c96663eac0b1ab5fa&submission=false" elapsedTime=1.819935ms httpStatus=200 remoteAddr=93.200.208.10 requestId=fe918e0edfb1/CBFxgx1qQ9-000022 responseSize=110 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:58:00Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/rest/stream?u=mg&t=30f3daf29af6381329d5e6abd40837c4&s=373128&f=json&v=1.8.0&c=NavidromeUI&id=636eed0b1b989c5c96663eac0b1ab5fa&_=1611251882636" elapsedTime=510.082893ms httpStatus=206 remoteAddr=93.200.208.10 requestId=fe918e0edfb1/CBFxgx1qQ9-000021 responseSize=12151336 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:58:02Z" level=trace msg="SQL: `SELECT * FROM user WHERE user_name LIKE ?`" args="['mg']" elapsedTime="194.031µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000023 rowsAffected=1
time="2021-01-21T17:58:02Z" level=trace msg="SQL: `SELECT starred, starred_at, play_count, play_date, rating, * FROM album LEFT JOIN annotation on (annotation.item_id = album.id AND annotation.item_type = 'album' AND annotation.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') ORDER BY  created_at desc LIMIT 12`" args="[]" elapsedTime=21.18567ms requestId=fe918e0edfb1/CBFxgx1qQ9-000023 rowsAffected=12
time="2021-01-21T17:58:02Z" level=trace msg="SQL: `SELECT starred, starred_at, play_count, play_date, rating, *, count(*) as count FROM album LEFT JOIN annotation on (annotation.item_id = album.id AND annotation.item_type = 'album' AND annotation.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e')`" args="[]" elapsedTime=8.33719ms requestId=fe918e0edfb1/CBFxgx1qQ9-000023 rowsAffected=1
time="2021-01-21T17:58:02Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/app/api/album?_end=12&_order=DESC&_sort=created_at&_start=0" elapsedTime=30.250894ms httpStatus=200 remoteAddr=93.200.208.10 requestId=fe918e0edfb1/CBFxgx1qQ9-000023 responseSize=13160 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:58:06Z" level=trace msg="SQL: `SELECT * FROM user WHERE user_name LIKE ?`" args="['mg']" elapsedTime="173.677µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000024 rowsAffected=1
time="2021-01-21T17:58:06Z" level=trace msg="SQL: `SELECT starred, starred_at, play_count, play_date, rating, * FROM album LEFT JOIN annotation on (annotation.item_id = album.id AND annotation.item_type = 'album' AND annotation.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') WHERE ((full_text LIKE ?)) ORDER BY  created_at desc LIMIT 12`" args="['% wintergatan%']" elapsedTime=6.230175ms requestId=fe918e0edfb1/CBFxgx1qQ9-000024 rowsAffected=4
time="2021-01-21T17:58:06Z" level=trace msg="SQL: `SELECT starred, starred_at, play_count, play_date, rating, *, count(*) as count FROM album LEFT JOIN annotation on (annotation.item_id = album.id AND annotation.item_type = 'album' AND annotation.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') WHERE ((full_text LIKE ?))`" args="['% wintergatan%']" elapsedTime=5.947425ms requestId=fe918e0edfb1/CBFxgx1qQ9-000024 rowsAffected=1
time="2021-01-21T17:58:06Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/app/api/album?_end=12&_order=DESC&_sort=created_at&_start=0&name=wintergatan" elapsedTime=12.867147ms httpStatus=200 remoteAddr=93.200.208.10 requestId=fe918e0edfb1/CBFxgx1qQ9-000024 responseSize=4219 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:58:07Z" level=trace msg="Broker received new event" event="{8553 keepAlive {\"ts\":1611251887}}"
time="2021-01-21T17:58:08Z" level=trace msg="SQL: `SELECT * FROM user WHERE user_name LIKE ?`" args="['mg']" elapsedTime="171.872µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000026 rowsAffected=1
time="2021-01-21T17:58:08Z" level=trace msg="SQL: `SELECT starred, starred_at, play_count, play_date, rating, media_file.*, position as bookmark_position FROM media_file LEFT JOIN annotation on (annotation.item_id = media_file.id AND annotation.item_type = 'media_file' AND annotation.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') LEFT JOIN bookmark on (bookmark.item_id = media_file.id AND bookmark.item_type = 'media_file' AND bookmark.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') WHERE (album_id = ?) ORDER BY disc_number asc, track_number asc`" args="['080be746d429c317375f89037cf4a70b']" elapsedTime="900.51µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000026 rowsAffected=9
time="2021-01-21T17:58:08Z" level=trace msg="SQL: `SELECT starred, starred_at, play_count, play_date, rating, count(*) as count FROM media_file LEFT JOIN annotation on (annotation.item_id = media_file.id AND annotation.item_type = 'media_file' AND annotation.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') WHERE (album_id = ?)`" args="['080be746d429c317375f89037cf4a70b']" elapsedTime="146.959µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000026 rowsAffected=1
time="2021-01-21T17:58:08Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/app/api/song?_end=0&_order=ASC&_sort=discNumber%20asc%2C%20trackNumber%20asc&_start=0&album_id=080be746d429c317375f89037cf4a70b" elapsedTime=1.688371ms httpStatus=200 remoteAddr=93.200.208.10 requestId=fe918e0edfb1/CBFxgx1qQ9-000026 responseSize=10343 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:58:08Z" level=trace msg="SQL: `SELECT * FROM user WHERE user_name LIKE ?`" args="['mg']" elapsedTime="220.343µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000027 rowsAffected=1
time="2021-01-21T17:58:08Z" level=trace msg="SQL: `SELECT starred, starred_at, play_count, play_date, rating, * FROM album LEFT JOIN annotation on (annotation.item_id = album.id AND annotation.item_type = 'album' AND annotation.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') WHERE id = ?`" args="['080be746d429c317375f89037cf4a70b']" elapsedTime="673.252µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000027 rowsAffected=1
time="2021-01-21T17:58:08Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/app/api/album/080be746d429c317375f89037cf4a70b" elapsedTime=1.26313ms httpStatus=200 remoteAddr=93.200.208.10 requestId=fe918e0edfb1/CBFxgx1qQ9-000027 responseSize=975 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:58:12Z" level=debug msg="API: New request /mg/rest/stream" client=NavidromeUI requestId=fe918e0edfb1/CBFxgx1qQ9-000028 username=mg version=1.8.0
time="2021-01-21T17:58:12Z" level=trace msg="SQL: `SELECT * FROM user WHERE user_name LIKE ?`" args="['mg']" elapsedTime="225.647µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000028 rowsAffected=1
time="2021-01-21T17:58:12Z" level=trace msg="playerId found in cookies" playerId=86b378d4-9a05-41fb-a55a-2e7625ea9a85 requestId=fe918e0edfb1/CBFxgx1qQ9-000028
time="2021-01-21T17:58:12Z" level=trace msg="SQL: `SELECT * FROM player WHERE id = ?`" args="['86b378d4-9a05-41fb-a55a-2e7625ea9a85']" elapsedTime="93.381µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000028 rowsAffected=1
time="2021-01-21T17:58:12Z" level=trace msg="SQL: `UPDATE player SET client = ?, id = ?, ip_address = ?, last_seen = ?, max_bit_rate = ?, name = ?, report_real_path = ?, transcoding_id = ?, type = ?, user_name = ? WHERE id = ?`" args="['NavidromeUI','86b378d4-9a05-41fb-a55a-2e7625ea9a85','','2021-01-21T17:58:12.30647257Z',0,'NavidromeUI (mg)',false,'','Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36','mg','86b378d4-9a05-41fb-a55a-2e7625ea9a85']" elapsedTime="179.849µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000028 rowsAffected=1
time="2021-01-21T17:58:12Z" level=trace msg="SQL: `SELECT starred, starred_at, play_count, play_date, rating, media_file.*, position as bookmark_position FROM media_file LEFT JOIN annotation on (annotation.item_id = media_file.id AND annotation.item_type = 'media_file' AND annotation.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') LEFT JOIN bookmark on (bookmark.item_id = media_file.id AND bookmark.item_type = 'media_file' AND bookmark.user_id = '52fb8927-80e1-40ee-b1a6-e3b82efd661e') WHERE id = ?`" args="['9fad2d65ebd858a51ec0c9f735d3e5c4']" elapsedTime="589.516µs" requestId=fe918e0edfb1/CBFxgx1qQ9-000028 rowsAffected=1
time="2021-01-21T17:58:12Z" level=debug msg="Streaming RAW file" id=9fad2d65ebd858a51ec0c9f735d3e5c4 originalBitrate=830 originalFormat=flac path="/music/W/Wintergatan - Wintergatan/Slottskogen Disc Golf Club_Z4434.flac" requestBitrate=0 requestFormat= requestId=fe918e0edfb1/CBFxgx1qQ9-000028 selectedBitrate=0 selectedFormat=raw
time="2021-01-21T17:58:12Z" level=info msg="Streaming file" artist=Wintergatan bitRate=0 cached=false format=raw originalBitRate=830 originalFormat=flac title="Slottskogen Disc Golf Club" transcoding=false user=mg
time="2021-01-21T17:58:16Z" level=debug msg="HTTP: GET http://audio.XXXX.net/mg/rest/stream?u=mg&t=30f3daf29af6381329d5e6abd40837c4&s=373128&f=json&v=1.8.0&c=NavidromeUI&id=9fad2d65ebd858a51ec0c9f735d3e5c4&_=1611251894949" elapsedTime=3.730050591s httpStatus=206 remoteAddr=93.200.208.10 requestId=fe918e0edfb1/CBFxgx1qQ9-000028 responseSize=17967424 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.142 Safari/537.36"
time="2021-01-21T17:58:22Z" level=trace msg="Broker received new event" event="{8554 keepAlive {\"ts\":1611251902}}"```

**Comment by quotengrote** [comment]
> Unfortunately that download link doesn't seem to work?

That should work:
https://mgrote.next-cloud.org/index.php/s/wGsqWHezZQSdgs9

**Comment by deluan** [comment]
Sorry for the delay. I tried your file locally, and it seems that it is not playable in Chrome, so the Navidrome UI cannot play the file. Don't know why it is not playable.... There's nothing much Navidrome can do if the browser can't play the file. One possible workaround is to enable transcoding in Navidrome. I tried here and it was able to play the file.

I'll close this, but feel free to add any other concerns.
